<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget parent="main_window" name="Main Window">
  <Tab containers="1" tab_name="tab1">
   <Container>
    <DockSplitter orientation="-" sizes="0.50073;0.49927" count="2">
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range left="-37.840000" right="-4.284000" bottom="-3.541524" top="2.165224"/>
       <limitY/>
       <curve color="#1f77b4" name="/iris_0/imu_gazebo/angular_velocity/x"/>
       <curve color="#d62728" name="/iris_0/imu_gazebo/angular_velocity/y"/>
       <curve color="#1ac938" name="/iris_0/imu_gazebo/angular_velocity/z"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range left="-37.840000" right="-4.284000" bottom="-7.658399" top="17.432786"/>
       <limitY/>
       <curve color="#1f77b4" name="/iris_0/imu_gazebo/linear_acceleration/x"/>
       <curve color="#d62728" name="/iris_0/imu_gazebo/linear_acceleration/y"/>
       <curve color="#1ac938" name="/iris_0/imu_gazebo/linear_acceleration/z"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <Tab containers="1" tab_name="tab2">
   <Container>
    <DockSplitter orientation="-" sizes="1" count="1">
     <DockArea name="...">
      <plot style="Lines" mode="TimeSeries">
       <range left="0.084000" right="3.184000" bottom="-0.003008" top="0.128697"/>
       <limitY/>
       <curve color="#1f77b4" name="/iris_0/perf_analyzer/instant_power/data"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="1"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="MQTT Subscriber"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin status="idle" ID="ROS /rosout Visualization"/>
  <plugin status="idle" ID="ROS Topic Re-Publisher"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

